docs/source/isaura.rst

```diff
@@ -1,5 +1,160 @@
+     - ``int``
+     - Maximum number of hits for an event to be processed.
+
+
+.. _Isaura workflow:
+
+Workflow
+--------
+
+The basic idea behind Isaura is to run the "traditional" *Paolina* algorithm over the deconvoluted hits output in :doc:`beersheba`, in order to convert ``events`` into ``tracks`` with ``blobs``.
+
+The first step within the algorithm consists in checking that the number of hits is lower than the value provided in the config file (``max_num_hits``). That argument was introduced because, when running *Paolina* algorithm after :doc:`penthesilea`, there were some events that comprise such large amount of hits that the tracking information extraction took ridiculously long. As the plot presented below shows [#]_, high energy (trigger2) events usually contain around 200 *Penthesilea hits*, while there are some of them with more than 10000 hits.
+
+ .. image:: images/isaura/nhits_per_evt_r8571.jpg
+   :width: 1000
+
+The plot also shows that these events only  appear a few times within a 24h-long low-background run (around 0.05% of the total set of events). Rejecting these type of events is not a particularly worrysome issue: they would be thrown away in the posterior analysis, seeing that none of them are exclusively contained inside the fiducial volume. The spatial distributions for one of these events is presented below (where the grey dashed lines illustrate the boundaries of the chamber):
+
+
+ .. image:: images/isaura/XY_Z_distributions_evt_many_hits.jpg
+   :width: 1000
+
+In any case, one can easily infer from the plots that these events are not physical. On the contrary, they seem to correspond to either some kind of flash occurring in the chamber (like a mini-spark) or some fail in the electronics (after the saturation of an alpha particle, for example). The ID of the events that are removed from the reconstruction chain because of this reason will be specified in the table ``Filters/topology_select``, in order to keep track of this information.
+
+Another obvious condition that must fulfill all events to be processed is to contain hits. If not, the event will be also rejected, which will be displayed in the table ``Filters/hits_select``.
+
+..
+ Next step includes another (quite obvious) check: at least one hit inside the event must have a well-defined energy. If not, the event will be also rejected, since no topological information could be extracted.
+
+
+.. _Exracting the topology-related information:
+
+Extracting the topology-related information
+:::::::::::::::::::::::::::::::::::::::::::
+As it is well known, an excellent topological discrimation between signal and background (thanks to the usage of a gaseous medium inside the TPC) is one of the fundamental trademarks of the NEXT experiment. In order to do that, it is necessary to (1) separate the different tracks that may form the event, (2) find the extremes for each of them, and (3) compute the energy around these extremes, providing the so-called *blobs*.
```

Comment:
`In order to do that` --> In order to achieve that. You are referring to the topological discrimination, which you don't _do_, but rather _achieve_.